iTunesSearch/ItunesSearch.m

```diff
@@ -96,6 +97,10 @@ - (void)performApiCallForMethod:(NSString*)method
     if (self.affiliateToken && self.affiliateToken.length > 0)
         newParams[@"at"] = self.affiliateToken;
 
+    // Add campaign token if supplied
+    if (self.campaignToken && self.campaignToken.length > 0)
```

Comment:
I see now that curly braces was probably supposed to be used there to. I'' make a new commit.